site/_docs/reference.md

```diff
@@ -2693,6 +2693,8 @@ BigQuery's type system uses confusingly different names for types and functions:
 | b o | GREATEST(expr [, expr ]*)                    | Returns the greatest of the expressions
 | b h s | IF(condition, value1, value2)              | Returns *value1* if *condition* is TRUE, *value2* otherwise
 | b | IFNULL(value1, value2)                         | Equivalent to `NVL(value1, value2)`
+| b o | INSTR(string, substring[, from, occurrence]) | Returns the position of substring in string, searching starting at from, and until locating the nth occurrence of substring.
```

Comment:
s/substring/\*substring\*/ etc.

s/from/from (default 1)/

s/occurrence/occurrence (default 1)/

remove '.' at end of line